config/applets/color_mesh.conf

```diff
@@ -13,11 +14,14 @@ all_frames = false
 
 # We compare the depth buffer value with the depth of the mesh point.
 # We use threshold >= 0 to fix floating point inaccuracies
-# Default value is 0, bigger values will remove more points.
-occlusion_threshold = 0.0
+# Default value is 1, bigger values will remove more points.
+occlusion_threshold = 1
 
-# Remove occluded points if parameter is true.
+# Do not color occluded points if parameter is true.
 remove_occluded = true
 
-# Remove masked points if parameter is true.
+# Do not color masked points if parameter is true.
 remove_masked = true
+
+# Remove points that are not colored.
+remove_not_colored = false
```

Comment:
@danlipsa I'm wondering if we should make this a threshold instead of just a binary option.  If that value is `0` you get the current remove if not colored behavior.  If _N_ > 0 you remove frames with _N_ or less observations.  Then maybe `-1` could be a special case meaning don't remove anything.